### PR TITLE
Launch.java:68-70: Add new section to the... #69

### PR DESCRIPTION
**Pull request by pinaf**
Launch.java:68-70: Add new section to the... #69

### CODE REVIEW

**Comment by karato** [comment]
Thanks, I will find someone to review this PR soon

**Comment by karato** [comment]
@alevohin review it

File: README.md [hunk 1]
```diff
@@ -29,6 +29,22 @@ You might want to start this app on different port, just add port number as para
...
+### Download
+```GET https://nerodesk.com/api/file/$PATH```
+
+The file contents will be returned in the response's body.
+
+### Upload
+```PUT https://nerodesk.com/api/file/$PATH```
+
+The file contents should be present in the request's body.
+
```
Comment (alevohin) [inline README.md:48]: Please add `Delete` section. Puzzle mentions about it

> GET, PUT and DELETE operations should be described....

**Comment by alevohin** [comment]
@pinaf thank you! Please see one comment above.

File: README.md (hunk 1 above)
Comment (pinaf) [inline README.md:48]: @alevohin not sure why it's not there. probably lost it in copy/paste, sorry.

**Comment by pinaf** [comment]
@alevohin fixed

**Comment by alevohin** [comment]
@pinaf thank you! Sorry but I suggest to do one more thing - lets add mention about `404 if res. not available` (puzzle has no mentions about it, please se https://github.com/teamed/nerodesk/issues/14).

**Comment by pinaf** [comment]
@alevohin thanks. please take another look.

**Comment by alevohin** [comment]
@rultor merge @pinaf thank you, looks good!

**Comment by alevohin** [comment]
@karato @pinaf lets wait for https://github.com/teamed/nerodesk/pull/77 because it's not clear about API (see https://github.com/teamed/nerodesk/pull/77#issuecomment-80655843 - API has changed).

**Comment by pinaf** [comment]
@yegor256  @alevohin so what happens now?

**Comment by yegor256** [comment]
@pinaf it's out dated. the task will be closed/paid since I removed the puzzle from the code. but this pull request is obsolete

**Comment by pinaf** [comment]
@yegor256 Alright. But keep in mind there is no description in readme for the API.

**Comment by alevohin** [comment]
@pinaf could you close this PR?

**Comment by pinaf** [comment]
@alevohin yup. thanks.

**Comment by karato** [comment]
@alevohin **19 mins** was added to your account, many thanks for your contribution!

extra minutes for review comments (c=4)

added +19 to your rating, now it is equal to [+1676](http://www.netbout.com/b/35105?open=rating)

**Pull request closed without merging**